### ISSUE TITLE
the size of test set

### ISSUE THREAD

**Issue by zhijm** [zhuoyunli/iCaRL#10]
Please. why the number of test sets is more than that of training sets in the training process?
for instance
the size of train set is (6980, 32, 32, 3)
the size of train label is (6980,)
the size of test set is (10000, 32, 32, 3)
the size of test label is (10000,)
please.
